Cookbook.Infrastructur/InfrastructureServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Cookbook.Application.Database;
+using Cookbook.Infrastructur.Persistance;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Cookbook.Infrastructur
```

Comment:
Ez az Infrastructure nev is eleg fura nekem, foleg igy hogy ekkora a project. 
Infrastructure-aba bele tartoznak a serverek, az adatbazisok, a message queue-k, de az inkabb a "hosting" contextusban letezik, mi operations soran beszelunk infrastructure-rol, hogy hany hostot kell patch-elnunk, meg hogy kell uj host-okat rendelnunk.

Ha lenne message queue-d meg SNS meg nem tudom mi, akkor meg lehet mukodne a nev, de igy inkabb hivnam en Database-nek vagy Repositories-nak

Reply:
hey hey hey its Infrastructur

Reply:
okay makes sense